ntbs-service/DataAccess/AlertRepository.cs

```diff
@@ -139,10 +139,12 @@ private IQueryable<AlertWithTbServiceForDisplay> GetOpenAlertWithTbServiceForDis
                             : alert.Notification.HospitalDetails.TBServiceCode,
                         TbServiceName = alert is TransferAlert
                             ? ((TransferAlert)alert).TbService.Name
-                            : alert.Notification.HospitalDetails.TBServiceName,
+                            : alert.Notification.HospitalDetails.TBService != null
+                                ? alert.Notification.HospitalDetails.TBService.Name : null,

```

Comment:
I _think_ that because this is still inside SQL you can actually just write `alert.Notification.HospitalDetails.TBService.Name` here and it will handle null values for you.

Reply:
It does!